app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -34,12 +34,14 @@ def process_branch
   end
 
   def rubocop_comments
-    MessageBuilder.new(results, branch).comments
+    message_builder = MessageBuilder.new(results, branch)
+    @status = message_builder.commit_status
```

Comment:
I don't think this is *The Right Way&trade;* to pass the information, the `rubocop_comments` method is being used just in the `replace_rubocop_comments` method, so it's safe to move its content there. This way you don't need the instance variable at all. Then you can refactor or even rename the method and maybe break it into small(er) chunk.